integration/networktest/tests/nodescenario/restart_validator_test.go

```diff
@@ -0,0 +1,48 @@
+package nodescenario
+
+import (
+	"testing"
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest/actions"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+	"github.com/obscuronet/go-obscuro/integration/networktest/env"
+)
+
+// restart both the host and the enclave for a validator
+func TestRestartValidatorNode(t *testing.T) {
+	networktest.TestOnlyRunsInIDE(t)
+	networktest.Run(
+		"restart-node",
+		t,
+		env.LocalDevNetwork(),
+		actions.Series(
+			actions.CreateAndFundTestUsers(5),
+
+			// short load test, build up some state
+			actions.GenerateUsersRandomisedTransferActionsInParallel(4, 10*time.Second),
+
+			actions.SleepAction(10*time.Second), // allow time for in-flight transactions
```

Comment:
quick note here. 
not for this pr, but maybe a todo.
This sleep could be replaced with an action that reads from the context all transaction ids generated earlier and polls for them.